website/docs/docs/explore/navigate-dbt-insights.md

```diff
@@ -24,31 +24,24 @@ The query console is the main component of <Constant name="query_page" />. It al
 - Query console editor, which allows you to write, run, and analyze SQL queries:
   - It supports syntax highlighting and autocomplete suggestions 
   - Hyperlink from SQL code `ref` to the corresponding Explorer page
-- [Query console menu](#query-console-menu), which contains **Bookmark** and **Run** buttons. 
-  - ✨ Coming soon: Ability to open the [<Constant name="cloud_ide" />](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [<Constant name="visual_editor" />](/docs/cloud/canvas) to continue editing your SQL query.
+- [Query console menu](#query-console-menu), which contains **Bookmark (icon)**, **Develop**, and **Run** buttons. 
 - [Query output panel](#query-output-panel), below the query editor and displays the results of a query:
   - Has three tabs: **Results**, **Details**, and **Chart**, which allow you to analyze query execution and visualize results.
 - [Query console sidebar menu](#query-console-sidebar-menu), which contains the **<Constant name="explorer" />**, **Bookmark**, **Query history**, and **<Constant name="copilot" />** icons.
 
 <Lightbox src="/img/docs/dbt-insights/insights-main.png" title="dbt Insights main interface with blank query editor" />
```

Comment:
this looks like we may need to redo with the brand refresh!

Reply:
branch refresh screenshots can come after the release, @matthewshaver and @dave-connors-3